### DISCUSSION TITLE
Specify Font Size per Font

### DISCUSSION THREAD

**Discussion by chxp82q** [wezterm/wezterm#715] [Q&A] [answered]
I use symbols from a package that just gives nerd font symbols - and it's too big. Is there a way I can specify the font size for a fallback font or change the default size?

**Comment by wez** [comment, marked as answer]
In theory, since https://github.com/wez/wezterm/commit/3fae59b01ba1cbd991e60e1b7ae9bfa0ac449aa2, wezterm should be able to do a reasonable job at scaling glyphs automatically.  I did just push a fix where we could end up with a size that is too small... would you mind sharing the config where you see the wrong size?

wezterm doesn't currently have a configuration that can specify the size per font, but could potentially evolve one.  I'd prefer to get the cap-height heuristics in better shape as that theoretically means that things are more likely to "just work"!
